core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/PillarChangeMove.java

```diff
@@ -55,9 +55,8 @@ public boolean isMoveDoable(ScoreDirector<Solution_> scoreDirector) {
         }
         if (!variableDescriptor.canExtractValueRangeFromSolution()) {
             var valueRangeDescriptor = variableDescriptor.getValueRangeDescriptor();
-            var workingSolution = scoreDirector.getWorkingSolution();
             for (Object entity : pillar) {
-                var rightValueRange = extractValueRange(scoreDirector, valueRangeDescriptor, workingSolution, entity);
+                var rightValueRange = extractValueRange(scoreDirector, valueRangeDescriptor, null, entity);
```

Comment:
This only confirms my earlier review comment - if the two calls were separate for entity and for solution, this wouldn't have happened.